TOOLS/osxbundle/mpv.app/Contents/Info.plist

```diff
@@ -796,13 +992,92 @@
         <key>UTTypeIdentifier</key>
         <string>io.mpv.sub</string>
         <key>UTTypeReferenceURL</key>
-        <string>http://en.wikipedia.org/wiki/MicroDVD</string>
+        <string>https://en.wikipedia.org/wiki/MicroDVD</string>
         <key>UTTypeTagSpecification</key>
         <dict>
           <key>public.filename-extension</key>
           <array>
             <string>sub</string>
           </array>
+          <key>public.mime-type</key>
+          <array>
+            <string>text/plain</string>
+          </array>
+        </dict>
+      </dict>
+      <dict>
+        <key>UTTypeConformsTo</key>
+        <array>
+          <string>public.plain-text</string>
+        </array>
+        <key>UTTypeDescription</key>
+        <string>AQTitle Subtitle</string>
+        <key>UTTypeIconFile</key>
+        <string>document.icns</string>
+        <key>UTTypeIdentifier</key>
+        <string>io.mpv.aqt</string>
+        <key>UTTypeReferenceURL</key>
+        <string>https://ffmpeg.org/doxygen/4.1/aqtitledec_8c_source.html</string>
```

Comment:
don't think we should use links to the ffmpeg docs here since they versioned. eg this one links to the version in ffmpeg 4.1 and were are at 7.0 i believe.